stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -74,6 +74,9 @@ spec:
             chmod 700 {{ .Values.persistence.mountPath }}/data
             find {{ .Values.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | \
               xargs chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}
+            if [ -d {{ .Values.persistence.mountPath }}/data ]; then
+              chmod  0700 {{ .Values.persistence.mountPath }}/data;
```

Comment:
Aren't we already changing the permissions of the data folder a couple of lines above?